Makefile

```diff
@@ -723,10 +723,16 @@ build-book: ## Build the book
 
 ##@ test:
 
+VSCODE_TEST_ENV_FILE ?= ${ROOT_DIR}/.vscode/test.env
```

Comment:
Q: Where is this var used?

Reply:
Unused, accidentally left over from earlier work. Removing. Thanks!